config/logger.js

```diff
@@ -9,8 +9,18 @@ const config = {
   },
 };
 
-const logger = PinoLogger.pino(
-  process.env.NODE_ENV === "development" ? config : {}
-);
+let logger
+
+if (process.env.STORYBOOK_RUN) {
+  logger = {
+    child: console.log,
+    info: console.log,
+    error: console.log,
```

Comment:
By default I think Pino it goes to standard out, would this still be needed?

Reply:
Ah interesting, I selected Pino because it is meant to work in the browser over something like Winston - ok no problem, we can always tweak in the future

Reply:
According to the [Pino Docs](https://github.com/pinojs/pino/blob/master/docs/browser.md), in order for it to work with the browser it needs to be used with [Browserify](https://github.com/browserify/browserify).
Browserify is a pre-processer that resolves all the requires & polyfills node globals (process, __filename, __dirname, etc).
This won't work easily with the Storybook setup, so the polyfill I used seems like a simpler solution.

BTW, the logger code never runs, because it is called from `getServerSideProps()` which isn't run by Storybook.
The component is rendered with the props specified in the story.

Reply:
ok thank you for the detailed reply Dan 👍